samples/xml/sportsml/team-roster.xml

```diff
@@ -53,7 +53,7 @@
         <inlineXML contenttype="application/sportsml+xml">
             <sports-content>
                 <statistic content-label="Team Roster">
-                    <statistic-metadata team-coverage="spteamcoverage:single-team"
+                    <statistic-metadata team-coverage="single-team"
```

Comment:
Hey @pauljkelly , I noticed that this change breaks the XSLT processing - line 423 of tools/sportsML-to-n3.xsl looks specifically for 'spteamcoverage:single-team' so either this needs to change back, or that line needs to change to remove the qcode prefix. Which would you prefer?

Reply:
For posterity, we changed the XSLT.